src/rustdoc.md

```diff
@@ -34,6 +34,54 @@ does is call the `main()` that's in this crate's `lib.rs`, though.)
 
 [bin]: https://github.com/rust-lang/rust/tree/master/src/tools/rustdoc
 
+## Fast builds
+
+Preconfigure your `/config.toml`:
+
+```toml
```

Comment:
This is all way overkill. I would suggest `x.py setup tools` and nothing else.

Reply:
I had "libraries" configured  before and the setup would not touch the file anymore, so I decided to do it by hand. Is this intended behaviour?

Reply:
Ok, then I would recommend `profile = "tools"` instead - that applies the latest changes automatically and also means you don't need to copy/paste 20 lines of config.

Reply:
Are the implications for the "tools" profile documented somewhere? I would like to link to it

Reply:
All the profiles are documented in bootstrap itself, or otherwise it would get out of date: https://github.com/rust-lang/rust/blob/master/src/bootstrap/defaults/config.tools.toml